Sources/App/Core/Constants.swift

```diff
@@ -31,8 +31,7 @@ enum Constants {
     static let rssFeedMaxItemCount = 100
     static let rssTTL = 60  // minutes
     
-    static let searchLimit = 20
-    static let searchLimitLeeway = 5
```

Comment:
I've had to drop the "leeway" mechanism as that was too hard to reconcile with paging.

Reply:
Yea, I'd say this is no longer needed.